frontend/src/index.html

```diff
@@ -6,91 +6,140 @@
+                    <path
+                      fill="white"
+                      d="M8 6.82v10.36c0 .79.87 1.27 1.54.84l8.14-5.18c.62-.39.62-1.29 0-1.69L9.54 5.98C8.87 5.55 8 6.03 8 6.82z"
+                    />
+                  </svg>
+                </button>
+              </div>
+              <div class="col-span-5 content-center pt-1 md:col-span-7">
+                <div class="truncate text-xs font-semibold">{{ audio.title }}</div>
+                <div class="mb-1 flex flex-wrap truncate text-xs [&>*]:mb-1 [&>*]:mr-1 [&>*]:border-r [&>*]:border-gray-300 [&>*]:pr-1 last:[&>*]:border-0">
+                  <div>#{{ audio.objectID }}</div>
+                  {% if audio.dateForHumans -%}
+                    <div>
+                      {{ audio.dateForHumans }}
+                      {% if audio.dateUncertain -%}
+                        <span title="Date is uncertain">(?)</span>
+                      {%- endif %}
+                    </div>
                   {%- endif %}
-                </div>
-              {%- endif %}
-              {% if audio.location -%}
-                <div>
-                  {{ audio.location }}
-                  {% if audio.locationUncertain -%}
-                    <span title="Location is uncertain">(?)</span>
+                  {% if audio.location -%}
+                    <div>
+                      {{ audio.location }}
+                      {% if audio.locationUncertain -%}
+                        <span title="Location is uncertain">(?)</span>
+                      {%- endif %}
+                    </div>
+                  {%- endif %}
+                  {% if audio.category -%}
+                    <div>{{ audio.category }}</div>
                   {%- endif %}
+                  {% for language in audio.languages -%}
+                    <div>{{ language }}</div>
+                  {%- endfor %}
                 </div>
-              {%- endif %}
-              {% if audio.category -%}
-                <div>{{ audio.category }}</div>
-              {%- endif %}
-              {% for language in audio.languages -%}
-                <div>{{ language }}</div>
-              {%- endfor %}
+              </div>
             </div>
-          </div>
+          </a>
+        {% endfor %}
+      </div>
+    </div>
+  </div>
+  <!-- Body left ends -->
+  <!-- Body Right -->
+  <div class="px-2 sm:w-full md:w-3/12 md:!pl-2 md:!pr-4">
+    {% for dailyvideo in dailyVideos offset: 0 limit: 2 %}
+      <div class="mb-8 ">
+        <div class=" pb-2 !text-sm font-semibold uppercase text-black md:text-base">Today's harikatha ({{ dailyvideo.language | language_name }})</div>
+        <div
+          x-data="{showthumb: true}"
+          class="aspect-h-9 aspect-w-16"
+        >
+          <template x-if="showthumb">
+            <img
+              @click="showthumb = false"
+              class="rounded-md"
+              src="https://img.youtube.com/vi/{{ dailyvideo.videoID }}/hqdefault.jpg"
```

Comment:
In that StackOverflow post there was an answer that offer an alternative host name for getting the thumbnail. It was described there (maybe in comments) that using that alternative host might have some benefits. Please see one more time.